sdk/communication/azure-communication-chat/README.md

```diff
@@ -20,7 +20,7 @@ Azure Communication Chat contains the APIs used in chat applications for Azure C
 <dependency>
```

Comment:
I think we need to change this **Include the package** section to look more like Common's **Install the library**, where they also include how to add the package to your project using gradle.